apps/scripts/gpc_compile_link.sh

```diff
@@ -8,14 +8,14 @@ if [ "$1" == "-help" ];then
     cat ./README.txt
     exit 1
 fi
-str=riscv/install/rv32e/bin/
-mod=e
+str=riscv/install/rv32i/bin/
+mod=i
 if [ "$1" == "-e" ];then
     str=riscv_e/install/rv32e/bin/
     mod=e
 fi
 
-GNU_DIR=/home/saar/projects/$str
+GNU_DIR=/home/adlv/projects/$str
```

Comment:
How did you install RISC-V toolchain? It looks strange that you have two different places for rv32e and rv32i. I have a single toolchain for both, so it looks like a strange environment perk (which I hope is easy to solve)

Reply:
Saar and Adi are the 2 students. so i assume when Saar was running the script he would comment out the adlv path.

Probably a quick and dirty solution to some path problems they encounter.
Could be fix like should in other comments.